docs/user/howtos/gitjob_push.md

```diff
@@ -0,0 +1,104 @@
+One of the more interesting features of Rio was that it would let you set up a deployment that rebuilds and republishes when your code stored in Git is changed. 
+
+We can recreate this functionality using the GitJob CRD that's a part of [Rancher Fleet](https://fleet.rancher.io/).
+
+NOTE: We will improve this experience in the future!
+
+## Setup
+
+### Install GitJob
+
+If you don't have Rancher (or standalone Fleet) installed, we need to install the GitJob operator by following the isntructions found at https://github.com/rancher/gitjob#running.
+
+
+Then we need to setup the Service Account to run our Jobs with (since we don't need to do anything directly with the kube api, we don't need to add any role bindings to it):
+
+```
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: epinio-push
+```
+
+### Upload Epinio Config
+
+So the GitJob can authenticate and push correctly, we can upload our Epinio config file to the cluster with:
+
+```
+kubectl create secret generic epinio-creds --from-file=$HOME/.config/epinio/config.yaml
+```
+
+This will create a secret containing the config.yaml that was created locally when you do `epinio install` or `epinio config update`
```

Comment:
Alternatively, we can give permissions to the ServiceAccount to read the Epinio credentials secret and run `epinio config update` from the gitjob itself. This will mean, one less step in this setup. Otoh, it will need us to create a role with that permission and explain why, which probably makes the guide more complicated.

I think both ways are equal, complexity-wise

Reply:
Yeah, and I think something that is done client side is easier to document than something magic inside our platform 😄